app/models/bank_account.rb

```diff
@@ -1,7 +1,13 @@
 class BankAccount < PaymentMethod
   before_update :verify_account, unless: ->(bank_account) { bank_account.verified? }
 
-  attr_accessor :first_deposit, :second_deposit
+  attr_accessor :first_deposit, :second_deposit, :plaid_public_token, :plaid_account_id
+
+  def initialize(params = {})
+    @plaid_public_token = params[:plaid_public_token]
+    @plaid_account_id = params[:plaid_account_id]
```

Comment:
are these instance variables actually used anywhere? it seems like not.

Reply:
Oh cool! I hadn't realized that the params merge works with just the attr_accessors (for non-db-backed variables). I've now eliminated the initialize method entirely, so much cleaner. :)